src/win/tty.c

```diff
@@ -1615,28 +1615,16 @@ static int uv_tty_write_bufs(uv_tty_t* handle,
   /* We can only write 8k characters at a time. Windows can't handle much more
    * characters in a single console write anyway. */
   WCHAR utf16_buf[MAX_CONSOLE_CHAR];
```

Comment:
Maybe drop `MAX_CONSOLE_CHAR` and use a literal `8192` here since this is the only place left now that uses it.

Reply:
There's another occurrence of it I could instead replace with the named value. I guess it's not too significant either way though—since it's an obvious power-of-two, it's not a particularly mysterious magic number.

Reply:
I think `MAX_CONSOLE_CHAR ` should be kept.

Reply:
> There's another occurrence of it I could instead replace with the named value.

Where? I don't see any others:
```
$ git grep MAX_CONSOLE_CHAR refs/pr/1965:src/win/
refs/pr/1965:src/win/:tty.c:#define MAX_CONSOLE_CHAR 8192
refs/pr/1965:src/win/:tty.c:  WCHAR utf16_buf[MAX_CONSOLE_CHAR];
```

Reply:
It's `MAX_INPUT_BUFFER_LENGTH` and a literal `8192` in there